src/components/MapComponent.svelte

```diff
@@ -13,21 +14,30 @@
       style: 'mapbox://styles/caesarfia/clxzna0zz006q01p80zkl9v03'
     });
 
+    // example locations:
+
+    // interface Location {
+    //     latitude: number;
+    //     longitude: number;
+    // }
+
+    // const locations: Location[] = [
+    //     { latitude: 29.650201, longitude: -82.340019 }, 
+    //     { latitude: 29.648469, longitude: -82.349935 }, 
+    //     { latitude: 29.637931, longitude: -82.368317 },  
+    // ];
+
+    // locations.forEach(location => {
+    //     new mapboxgl.Marker()
+    //         .setLngLat([location.longitude, location.latitude])
+    //         .addTo(map);
+    // });
```

Comment:
```suggestion
```
Remove dead code before merging into main. Code snippets can always be recovered from git